src/value.c

```diff
@@ -310,6 +396,9 @@ void SIValue_Free(SIValue *v) {
 		case T_EDGE:
 			rm_free(v->ptrval);
 			return;
+		case T_ARRAY:
+			SIArray_Free(v->array);
```

Comment:
I think it would be more cleaner for `SIArray_Free` to take the full `SIValue v` as its argument, though I'm open to other opinions.

Reply:
done